index.html

```diff
@@ -103,9 +103,42 @@ <h2>A project for first-time contributions</h2>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
 
         <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` 
into `<!-- ________ Your_Name Contributor card START ________ -->`